lms/static/sass/shared/_alerts_pattern_library_shim.scss

```diff
@@ -19,38 +19,49 @@
 // #CONFIG
 // ----------------------------
 
-// alert colors
-$alert-information-color: #6fa0ba;
-$alert-warning-color: #fdbc56;
-$alert-error-color: #b20610;
-$alert-success-color: #25b85a;
-$alert-background-color: #fcfcfc;
-$alert-icon-color: #fcfcfc;
-$alert-border-grey: #cdd7db;
-$alert-shadow-grey: #eef1f2;
-
-// alert borders
-$alert-border-radius: 0.3125rem;
-$alert-border: 0.0625rem solid $alert-border-grey;
+// Overrides for other things available in the UXPL but not here
+// This is super gross. Don't do this in other places if you can avoid it.
+$palette-brand-accent: #2991c3;
```

Comment:
This is problematic for anyone who is going to theme these pages. I wonder if there's a cleaner way to be able to consume these colors.